### ISSUE TITLE
simple_conversion.py fails: 'TexEnv' object has no attribute 'all'

### ISSUE THREAD

**Issue by gvwilson** [alvinwan/TexSoup#67]
`simple_conversion.py` fails for all inputs:

command line:

```
$ python simple_conversion.py
LaTex file: test.tex
Traceback (most recent call last):
  File "simple_conversion.py", line 64, in <module>
    tex_dict = {"latex": {"contents": to_dictionary(TexSoup.TexSoup(tex_text).expr.all)}}
AttributeError: 'TexEnv' object has no attribute 'all'
```

test input:
```
\chapter{Introduction}

Grassroots \emph{groups} have sprung up around the world
```

**Comment by gvwilson** [comment]
Note that I am using the version of TexSoup currently available through pip (2019-05-19).

**Comment by alvinwan** [comment]
@gvwilson Thanks for reporting -- a contributor in the community has since fixed this #87
